test/test_maintainability_markdown_report.py

```diff
@@ -610,6 +610,28 @@ def testMakeLinksForUnitMetrics(self):
 
         self.assertEqual(table.strip(), inspect.cleandoc(expected).strip())
 
+    @mock.patch.dict(os.environ, {
+        "CI_SERVER_URL" : "https://example.com",
+        "CI_PROJECT_PATH" : "aap/noot",
+        "CI_COMMIT_REF_NAME" : "mybranch"
+    })
+    def testNormalizeLinksForSubsystems(self):
+        rc = self.toRefactoringCandidate(f"jan/aap.py::noot", "introduced", "UNIT_SIZE", "VERY_HIGH")
```

Comment:
Pedantic: `jan` is not part of our standard set of test data names. Consider `jet`, `mies` or `henk`. My first reaction now was: what's (our) Jan got to do with this.

Reply:
Changed it. But I did already previously use `jan` as a test name as an in-joke, which has already been discovered by the real @janlaan :)